src/main/java/com/minecolonies/colony/Structures.java

```diff
@@ -99,7 +99,7 @@ private static void loadStyleMaps(Path basePath) throws IOException
 
                 if (path.toString().endsWith(SCHEMATIC_EXTENSION))
                 {
-                    String filename = path.getFileName().toString().split("\\.schematic")[0];
+                    String filename = path.getFileName().toString().split("\\.nbt")[0];
                     String hut = filename.split("\\d+")[0];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'hut' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)